packages/kotti-ui/source/kotti-card/KtCard.vue

```diff
@@ -1,36 +1,64 @@
 <template>
-	<div :class="cardClass">
+	<div :class="cardClass" tabindex="0" @click.prevent="handleClick">
 		<div v-if="imgUrl" :class="imageRowClass">
-			<img class="kt-card__image-row__image" :src="imgUrl" />
+			<div v-if="isImgLoading" class="skeleton square md" />
+			<img v-else class="kt-card__image-row__image" :src="imgUrl" />
 		</div>
 		<div v-if="$slots['card-header']" class="kt-card__header">
-			<slot name="card-header" />
+			<div v-if="isTextLoading" class="skeleton rectangle md" />
+			<slot v-else name="card-header" />
 		</div>
 		<div v-if="$slots['card-body']" class="kt-card__body">
-			<slot name="card-body" />
+			<div v-if="isTextLoading" class="skeleton rectangle sm" />
+			<slot v-else name="card-body" />
 		</div>
-		<div v-if="$slots['card-footer']" class="kt-card__footer">
-			<slot name="card-footer" />
+		<div v-if="$slots['card-footer'] || hasActions" class="kt-card__footer">
```

Comment:
I though the `footer` slot was removed. Shouldn't it be just?
```suggestion
		<div v-if="hasActions" class="kt-card__footer">
```

Reply:
thanks you re right